packages/libs/src/services/identity/IdentityService.ts

```diff
@@ -125,10 +125,25 @@ export class IdentityService {
   }
 
   async setAuthFn(obj: Data) {
+    // get wallet from hedgehog and set as owner wallet
+    const ownerWallet = obj.wallet as Wallet
```

Comment:
should we change the type of `Data` instead of this cast here?

Reply:
actually yeah, we probably could